test/milton/MiltonConfiguration.t.sol

```diff
@@ -0,0 +1,178 @@
+        MiltonUsdc miltonUsdcImplementation = new MiltonUsdc();
+        ERC1967Proxy miltonUsdcProxy =
+        new ERC1967Proxy(address(miltonUsdcImplementation), abi.encodeWithSignature("initialize(bool,address,address,address,address,address)", false, address(usdc), address(usdc), address(usdc), address(usdc), address(usdc)));
+        IMiltonInternal miltonUsdc = IMiltonInternal(address(miltonUsdcProxy));
+        assertEq(miltonUsdc.getAsset(), address(usdc));
+    }
+
+    function testShouldCreateMiltonDai() public {
+        // when
+        TestERC20 dai = new TestERC20(2**255);
+        dai.setDecimals(18);
+        MiltonDai miltonDaiImplementation = new MiltonDai();
+        ERC1967Proxy miltonDaiProxy =
+        new ERC1967Proxy(address(miltonDaiImplementation), abi.encodeWithSignature("initialize(bool,address,address,address,address,address)", false, address(dai), address(dai), address(dai), address(dai), address(dai)));
+        IMiltonInternal miltonDai = IMiltonInternal(address(miltonDaiProxy));
+        assertEq(miltonDai.getAsset(), address(dai));
+    }
+
+    function testShouldRevertInitializerUsdtWhenMismatchAssetAndMiltonDecimals() public {
+        // when
+        TestERC20 usdt = new TestERC20(2**255);
+        usdt.setDecimals(8);
+        MiltonUsdt miltonUsdtImplementation = new MiltonUsdt();
+        vm.expectRevert(abi.encodePacked("IPOR_001"));
+        ERC1967Proxy miltonUsdtProxy =
+        new ERC1967Proxy(address(miltonUsdtImplementation), abi.encodeWithSignature("initialize(bool,address,address,address,address,address)", false, address(usdt), address(usdt), address(usdt), address(usdt), address(usdt)));
+        IMiltonInternal(address(miltonUsdtProxy));
+    }
+
+    function testShouldRevertInitializeUsdcWhenMismatchAssetAndMiltonDecimals() public {
+        // when
+        TestERC20 usdc = new TestERC20(2**255);
+        usdc.setDecimals(8);
+        MiltonUsdc miltonUsdcImplementation = new MiltonUsdc();
+        vm.expectRevert(abi.encodePacked("IPOR_001"));
+        ERC1967Proxy miltonUsdcProxy =
+        new ERC1967Proxy(address(miltonUsdcImplementation), abi.encodeWithSignature("initialize(bool,address,address,address,address,address)", false, address(usdc), address(usdc), address(usdc), address(usdc), address(usdc)));
+        IMiltonInternal(address(miltonUsdcProxy));
+    }
+
+    function testShouldRevertInitializerDaiWhenMismatchAssetAndMiltonDecimals() public {
+        // when
+        TestERC20 dai = new TestERC20(2**255);
+        dai.setDecimals(8);
+        MiltonDai miltonDaiImplementation = new MiltonDai();
+        vm.expectRevert(abi.encodePacked("IPOR_001"));
+        ERC1967Proxy miltonDaiProxy =
+        new ERC1967Proxy(address(miltonDaiImplementation), abi.encodeWithSignature("initialize(bool,address,address,address,address,address)", false, address(dai), address(dai), address(dai), address(dai), address(dai)));
+        IMiltonInternal(address(miltonDaiProxy));
```

Comment:
Do we need this line?

Reply:
No, but it will throw the warning that the variable above is not being used
![Screen Shot 2023-01-31 at 8 02 36 PM](https://user-images.githubusercontent.com/103587277/215857221-5af6469e-7c02-4a4d-baa4-2c29a01de30c.png)

Reply:
However you prefer, we can keep or remove

Reply:
I prefer to get clean code, a warning in the console could be